assetto-corsa-compconfig.json

```diff
@@ -0,0 +1,445 @@
+        "InputType":"number",
+        "MinValue":"-1",
+        "MaxValue":"100",
+        "IsFlagArgument":false,
+        "ParamFieldName":"racecraftRatingRequirement",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"-1",
+        "Placeholder":"-1 or 0 - 100",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Save Leaderboard",
+        "Category":"Server Settings",
+        "Description":"If enabled, leaderboards are saved to the \"results\" folder",
+        "Keywords":"leaderboard",
+        "FieldName":"dumpLeaderboards",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"dumpLeaderboards",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"0",
+        "EnumValues":{
+            "False":"0",
+            "True":"1"
+        }
+    },
+    {
+        "DisplayName":"Save Entry List",
+        "Category":"Server Settings",
+        "Description":"If enabled, entry lists are saved at the end of any qualifying session",
+        "Keywords":"dump,entry,list",
+        "FieldName":"dumpEntryList",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"dumpEntryList",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"0",
+        "EnumValues":{
+            "False":"0",
+            "True":"1"
+        }
+    },
+    {
+        "DisplayName":"Central Entry List Path",
+        "Category":"Server Settings",
+        "Description":"Sets a custom path to a central entry list, for sharing across servers",
+        "Keywords":"central,entry,list,path",
+        "FieldName":"centralEntryListPath",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"centralEntryListPath",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Track Selection",
+        "Category":"Gameplay Settings",
+        "Description":"Sets the track for the server",
+        "Keywords":"track",
+        "FieldName":"track",
+        "InputType":"enum",
+        "IsFlagArgument":false,
+        "ParamFieldName":"track",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"mount_panorama",
+        "EnumValues":{
+            "barcelona":"Barcelona",
+            "brands_hatch":"Brands Hatch",
+            "hungaroring":"Hungaroring",
+            "kyalami":"Kyalami",
+            "laguna_seca":"Laguna Seca",
+            "misano":"Misano",
+            "monza":"Monza",
+            "mount_panorama":"Mount Panorama",
+            "nurburgring":"Nurburgring",
+            "paul_ricard":"Paul Ricard",
+            "silverstone":"Silverstone",
+            "spa":"Spa",
+            "suzuka":"Suzuka",
+            "zolder":"Zolder",
+            "zandvoort":"Zandvoort",
+            "imola":"Imola",
+            "oulton_park":"Oulton Park",
+            "donington":"Donington",
+            "snetterton":"Snetterton"

```

Comment:
Perhaps alphabetize these?

Reply:
What was that comment about *my* OCD?!

Reply:
This isn't OCD. This is just basic organization. ;)

Reply:
Blame the devs. That's how they order them.

Reply:
We're better than the devs. We don't sink to their level.